pkg/policy/rule.go

```diff
@@ -134,22 +170,30 @@ func (r *rule) resolveL4Policy(ctx *SearchContext, state *traceState, result *L4
 
 	if !ctx.EgressL4Only {
 		for _, r := range r.Ingress {
-			found += mergeL4(ctx, "Ingress", r.ToPorts, result.Ingress)
+			cnt, err := mergeL4(ctx, "Ingress", r.ToPorts, result.Ingress)
+			if err != nil {
```

Comment:
As discussed on Slack, an error here should eventually lead to the port being blocked entirely.

Reply:
I think it's fine to do that in a follow-up though.